.github/workflows/build_and_push.yml

```diff
@@ -92,4 +92,12 @@ jobs:
           project_type: docker
 
       - name: Logout of Amazon ECR
-        run: docker logout ${{ steps.login-ecr.outputs.registry }}
\ No newline at end of file
+        run: docker logout ${{ steps.login-ecr.outputs.registry }}
+
+      - name: Deploy lambda
+        if: matrix.image == 'csp_violation_report_service/purge_stale_reports'
+        run: |
+          FNAME=$(awk -F  "/" '{print $2}' <<< "${{ matrix.image }}")
+          aws lambda update-function-code \
+            --function-name "$FNAME" \
+            --image-uri $REGISTRY/${{ matrix.image }}:latest > /dev/null 2>&1
```

Comment:
nit: could you change this to deploy the `$GITHUB_SHA` tagged image so we can see in the console which image is deployed.

Reply:
good call